internal/validators.js

```diff
@@ -36,6 +36,20 @@ function validateArray(value, name) {
   }
 }
 
+function validateStringArray(value, name) {
```

Comment:
For context which some readers will not be aware of, this file is a proxy for a file used widely in the Node.js codebase. Having matching files lets us write the same code to use in this repo and in PRs.

We might get asked not to add one-off extra validators to this file. But I think ok to have the code here for now!

Reply:
I will add a note for it